docs/2/technical/data-format/overview.md

```diff
@@ -172,7 +159,7 @@ Here the data has not been updated, so the `updaterId` is the same as the `creat
 },
 ```
 
- The information here is simply what we have just created. This is where you will find all the data that you integrate into Microshare® but we will see that later.
+This is the most important section of the data as it gives the information on what we have just created. This is where you will find the data that you integrated into Microshare® under that recType. This will be explored later on. 
```

Comment:
This section contains the core information as ingested by the Microshare Smart Network. The format of the data section will vary based on the recType. If the recType begins with io.microshare, then the cannonical format of this section will be documented in this section.